Meme-ify-Me/app/src/main/java/nyc/c4q/rosmaryfc/meme_ify_me/MainActivity.java

```diff
@@ -35,6 +35,9 @@ protected void onCreate(Bundle savedInstanceState) {
 
         Button cameraButton = (Button)findViewById(R.id.camera_button);
         cameraButton.setOnClickListener(cameraListener);
+
+        Button fromGalleryButton = (Button) findViewById(R.id.pic_from_gallery_button);
+        fromGalleryButton.setOnClickListener(GalleryListener);
```

Comment:
consistent logic for gallery listener and camera listener. looks good, I think we should merge this to master and test